lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj

```diff
@@ -93,16 +107,27 @@ under the License.
               LogStandardErrorAsError="true"
               Condition=" '$(OS)' == 'Windows_NT' " />
     </Target>
+    <Choose>
```

Comment:
Didn't another PR remove the idea of a `SolutionDir2`? Why do we need two? And if we do, can we document what it is for?